docs/running-in-docker.rst

```diff
@@ -39,10 +39,21 @@ official Locust docker image as a base image::
 
 
 Running a distributed load test on Kubernetes
-=============================================
+==============================================
 
-The easiest way to run Locust on Kubernetes is to use a `Helm chart <https://helm.sh/>`_.
+HELM
+*****
+
+One of the ways to run Locust on Kubernetes is to use a `Helm chart <https://helm.sh/>`_.
```

Comment:
This rephrasing of the statement is purely an opinion and  I fully understand if Locust want to state that HELM is the easiest way to run distributed tests. If the rephrasing is not okay, please let me know and I will restore the original phrasing. 

Opinion justification: 
HELM is an incredible tool and the provided chart does a lot to make sure that the processes is easy. However, the user is not really "shielded" from Kubernetes and have to be more aware many other moving peaces in order to get things right. That in my opinion makes it so the term "easiest" may cause a bit of misguidance.